validator/test/fixtures/assets/gke/data.json

```diff
@@ -105,7 +105,7 @@
           "https://www.googleapis.com/compute/v1/projects/pso-cicd8/zones/us-west1-b/instanceGroupManagers/gke-canary-west-default-pool-496ebc1d-grp"
         ],
         "legacyAbac": {
-          "enabled": true
+          "enabled": false
```

Comment:
We should probably have separate test fixtures/data for each rule.

Reply:
Created new test data for legacy abac only